examples/bert/yamls/test/glue.yaml

```diff
@@ -32,12 +32,12 @@ save_finetune_checkpoint_folder: ${save_finetune_checkpoint_prefix}/${base_run_n
 # Callbacks
 callbacks:
   lr_monitor: {}
-  speed_monitor: {}
+  # speed_monitor: {}
 
 # Scheduler
 scheduler:
   name: linear_decay_with_warmup
-  t_warmup: 0.06dur
+  t_warmup: 0.5dur
```

Comment:
is this intentional?

Reply:
Yeah, I made the change to prevent warnings about the warmup duration being 0 batches (which was happening with the test max duration of 10 batches).